services/beaconchain/src/main/java/tech/pegasys/artemis/services/beaconchain/BeaconChainController.java

```diff
@@ -169,7 +165,7 @@ public void initMetrics() {
 
   public void initValidatorCoordinator() {
     STDOUT.log(Level.DEBUG, "BeaconChainController.initValidatorCoordinator()");
-    this.validatorCoordinator = new ValidatorCoordinator(eventBus, chainStorageClient, config);
+    new ValidatorCoordinator(eventBus, chainStorageClient, config);
```

Comment:
Does this not make ValidatorCoordinator object prone to be deleted by garbage collector?

Reply:
No, `ValidatorCoordinator` registers with the event bus so as long as the event bus isn't GC'd it will hold a reference to `ValidatorCoordinator` and make it ineligible for GC.